tests/performance_tests/launch_transaction_generators.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import math
+import argparse
+import subprocess
+
+harnessPath = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(harnessPath)
+
+from TestHarness import Utils
+
+Print = Utils.Print
+
+tpsLimitPerGenerator = 1000
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument("chain_id", type=str, help="Chain ID")
+parser.add_argument("last_irreversible_block_id", type=str, help="Last irreversible block ID")
+parser.add_argument("handler_account", type=str, help="Cluster handler account name")
+parser.add_argument("account_1_name", type=str, help="First account name")
+parser.add_argument("account_2_name", type=str, help="Second account name")
+parser.add_argument("account_1_priv_key", type=str, help="First account private key")
+parser.add_argument("account_2_priv_key", type=str, help="Second account private key")
+parser.add_argument("trx_gen_duration", type=str, help="How long to run transaction generators")
+parser.add_argument("target_tps", type=int, help="Goal transactions per second")
+parser.add_argument("tps_limit_per_generator", type=int, help="Maximum amount of transactions per second a single generator can have.")
+args = parser.parse_args()
+
+targetTps = args.target_tps
+numGenerators = math.ceil(targetTps/args.tps_limit_per_generator)
+tpsPerGenerator = math.floor(targetTps/numGenerators)
+modTps = targetTps%numGenerators
+cleanlyDivisible = modTps == 0
+incrementPoint = numGenerators + 1 - modTps
+num_list = []
```

Comment:
Could we call this something more like `subprocesses` relating to the fact it is storing the return from `subprocess.Popen` and used to `.wait()`